### ISSUE TITLE
Add instructions for WAR-deployment on Heroku and OpenShift

### ISSUE THREAD

**Issue by brmeyer** [opendatakit/aggregate#104]
GAE has a lot of merits, but the Java PaaS ecosystem has come a long way in the last few years.  Heroku and OpenShift both make it exceedingly easy to deploy standard WARs, in addition to providing managed MySQL and Postgres instances.  The learning curve is quite a bit lower, and they're super cheap (Heroku's free or $7 plans would likely nail a lot of user's needs).

Regarding AWS, that's a toss up.  It's expensive and likely out-of-scope for orgs that do not have a devops or admin resource available.  There's probably merit in considering it, but maybe not until it becomes an actual need.

**Comment by ggalmazor** [comment]
Hi @brmeyer!

It doesn't feel like we're going to work on support for Heroku or OpenShift in the near future... Maybe @lognaturel can add something to this topic.

Anyway, if you (or anyone really) have had success in deploying to Heroku or OpenShift, I guess we could add it as a non-official guide in the docs. We could totally review a PR with that.

**Comment by yanokwa** [comment]
I think there is likely value in adding support for Heroku because free hosting remains a big requirement. Low cost is less so, because generally if you get a credit card to pay for $1, you can generally pay $50. For that reason, I'm adjusting this issue to Heroku only. It's still low priority, but it might be a nice thing for a contributor to take on.
